moped-editor/src/views/projects/projectView/ProjectComponents/CreateComponentDrawTools.js

```diff
@@ -11,13 +13,16 @@ import mapboxDrawStylesOverrides from "src/styles/mapboxDrawStylesOverrides";
  * @param {Function} setIsDrawing - function to update if we are drawing or not
  * @returns {JSX.Element}
  */
+
 const CreateComponentDrawTools = ({
   createDispatch,
   linkMode,
   setCursor,
   setIsDrawing,
 }) => {
   const drawControlsRef = useRef();
+  const [isTrashButtonClickable, toggleTrashButtonClickable] = useState(false);
```

Comment:
would you rename `toggleTrashButtonClickable` to start with `set`? just to be super clear its a state setter.